src/renderer/components/publication/TagManager.tsx

```diff
@@ -92,8 +103,10 @@ export class TagManager extends React.Component<Props, TagManagerState> {
     private addTag(e: any) {
         e.preventDefault();
         const { tags } = this.state;
-        if (this.state.nameNewTag && tags.indexOf(this.state.nameNewTag) < 0) {
-            tags.push(this.state.nameNewTag);
+        const currentTag = this.state.nameNewTag.trim();
```

Comment:
This seems to be fixing another issue, unrelated to the "enter key" UX. As the issue did not exist, I created it: #490
Can we please isolate these code changes into a separate Pull Request scope? Notably, we need to address inner-whitespace collapse as well (not just leading / trailing trim).